Make sure that the swupdate.service is not killed on OOM.

diff -aurN a/swupdate.service b/swupdate.service
--- a/swupdate.service	2021-03-09 10:49:04.782978190 +0100
+++ b/swupdate.service	2021-03-11 21:01:08.325024688 +0100
@@ -7,5 +7,8 @@
 ExecStart=@LIBDIR@/swupdate/swupdate.sh
 KillMode=mixed
 
+# Make sure service is not killed on OOM
+OOMScoreAdjust=-1000
+
 [Install]
 WantedBy=multi-user.target
